README.md

```diff
@@ -153,6 +153,10 @@ $ docker run --env-file .env.test todo-app:test
 
 Passing docker the relevant environment variables (--env-file) at runtime will help to keep your secrets safe, while also keeping your image re-usable - you can spin up multiple containers, each using different credentials.
 
+### Accessing the Application
+
+The Application is deployed to Azure and can be accessed via this link [https://jimmy-todo-app.azurewebsites.net/]
```

Comment:
Minor, but if you look at how this renders [e.g. here](https://github.com/jormondroyd27/DevOps-Course-Starter/tree/module-9) then you'll see it isn't actually formatting it at all (though it does pick up that it's a link)

You might be aiming for something more like:
```suggestion
The Application is deployed to Azure and can be [accessed via this link](https://jimmy-todo-app.azurewebsites.net/)
```